### PR TITLE
Removing CA derivations (to make Hercules CI work)

### PR DESCRIPTION
**Pull request by aciceri**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -199,6 +192,9 @@
               devShells.default = lib.mkDefault flk.devShell;
               project = prj;
             };
+            # TODO consider removing this since now we are using Hercules CI
```
Comment (aciceri) [inline flake.nix:195]: Tell me if you want that I remove this

File: mk-hackage.nix [hunk 2]
```diff
@@ -85,11 +85,10 @@ let
...
       '';
 
-    copySrc = src: pkgs.runCommand "copied-src-${builtins.baseNameOf src}" {
-      __contentAddressed = true;
-    } ''
-      cp -T -r ${src} $out
-    '';
+    copySrc = src: builtins.path {
+      path = src;
+      name = "copied-src-${builtins.baseNameOf (builtins.unsafeDiscardStringContext src)}";
```
Comment (aciceri) [inline mk-hackage.nix:90]: A bit puzzled on why `unsafeDiscardStringContext` is needed but passing a string with context as `name` was causing an errors like
```
the string 'copied-src-dxk8mk38rkq57c8qfcan4vg16c2jb6i4-source' is not allowed to refer to a store path (such as '')
```

File: flake.nix (hunk 1 above)
Comment (L-as) [reply flake.nix:195]: please remove

File: mk-hackage.nix (hunk 2 above)
Comment (L-as) [reply mk-hackage.nix:90]: maybe ask? try the other builtins?

Comment (aciceri) [reply mk-hackage.nix:90]: I'm asking you :)
do you mean other builtins instead of `path`?

Comment (L-as) [reply mk-hackage.nix:90]: yes

Comment (aciceri) [reply mk-hackage.nix:90]: mmm tbh `builtins.path` seems to me the right choice here, let discuss this on slack

**Pull request merged**